documentation/user-interface/test-editor.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+title: Test Editor
+permalink: /documentation/user-interface/test-editor.html
+---
+# Test Editor
```

Comment:
I think by analogy with previous one this section should be named `Test View`

Reply:
Let's use Test Editor and Fixture Editor